### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by andrew-kuzmin**

### CODE REVIEW

File: src/app/auth/register/register.component.html [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
+  <mat-card-content>
 <form [formGroup]="formControl" (ngSubmit)="register()">
   <div>
     <mat-form-field>
       <input matInput placeholder="First Name" formControlName="firstName">
-      <mat-error *ngIf="firstName.hasError('required')">
-        First name is <strong>required</strong>
-      </mat-error>
+      <mat-error *ngIf="firstName.hasError('required')">First name is <strong>required</strong></mat-error>
+<!--      <mat-error *ngIf="firstName.hasError( 'maxlength')">You have more than 50 characters</mat-error>-->
```
Comment (JackKarichkovskiy) [inline src/app/auth/register/register.component.html]: For removal?

File: src/app/auth/register/register.component.html [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
     </mat-form-field>
   </div>
   <div>
     <mat-form-field>
       <input matInput placeholder="Last Name" formControlName="lastName">
-      <mat-error *ngIf="lastName.hasError('required')">
-        Last name is <strong>required</strong>
-      </mat-error>
+      <mat-error *ngIf="lastName.hasError('required')">Last name is <strong>required</strong></mat-error>
+<!--      <mat-error *ngIf="firstName.hasError( 'maxlength')">You have more than 50 characters</mat-error>-->
```
Comment (JackKarichkovskiy) [inline src/app/auth/register/register.component.html]: For removal?

File: src/app/auth/register/register.component.html [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
   </div>
   <div>
     <mat-form-field>
-      <input matInput placeholder="Email" formControlName="email">
-      <mat-error *ngIf="email.hasError('required')">
-        Email is <strong>required</strong>
-      </mat-error>
+      <input matInput placeholder="E-mail" formControlName="email">
+      <mat-error *ngIf="email.hasError('required')">Email is <strong>required</strong></mat-error>
+<!--      <mat-error *ngIf="firstName.hasError( 'maxlength')">You have more than 50 characters</mat-error>-->
```
Comment (JackKarichkovskiy) [inline src/app/auth/register/register.component.html]: For removal?

File: src/app/auth/register/register.component.html [hunk 4]
```diff
@@ -1,51 +1,52 @@
...
     </mat-form-field>
   </div>
   <div>
     <mat-form-field>
       <input matInput placeholder="Phone number" formControlName="phone">
-      <mat-error *ngIf="phone.hasError('required')">
-        Phone number is <strong>required</strong>
-      </mat-error>
+      <mat-error *ngIf="phone.hasError('required')">Phone number is <strong>required</strong></mat-error>
+<!--      <mat-error *ngIf="firstName.hasError( 'maxlength')">You have more than 32 characters</mat-error>-->
```
Comment (JackKarichkovskiy) [inline src/app/auth/register/register.component.html]: For removal?

File: src/app/auth/register/register.component.html [hunk 5]
```diff
@@ -1,51 +1,52 @@
...
   </div>
   <div>
     <mat-form-field>
-      <input matInput placeholder="Rating" formControlName="rating">
-      <mat-error *ngIf="rating.hasError('required')">
-        Rating is <strong>required</strong>
-      </mat-error>
+      <input matInput type="number" placeholder="Rating" formControlName="rating">
+      <mat-error *ngIf="rating.hasError('required')">Rating is <strong>required</strong></mat-error>
+<!--      <mat-error *ngIf="firstName.hasError( 'maxlength')">Max value is 100</mat-error>-->
```
Comment (JackKarichkovskiy) [inline src/app/auth/register/register.component.html]: For removal?

Comment (andrew-kuzmin) [reply src/app/auth/register/register.component.html]: To be added in future

**Pull request merged**